### ISSUE TITLE
Add language codes (.en.srt) to subtitles stored next to media.

### ISSUE THREAD

**Issue by Kezxo** [bramwalet/Subliminal.bundle#30]
It's not doing that it seems.

**Comment by bramwalet** [comment]
Did you check your media folder? Does plex have permission to write to that folder? Did you also include LocalMedia(Extended) plugin and is the order of the plugin as per readme?

**Comment by Kezxo** [comment]
Hmm, looks like I was mistaken. Sorry. Maybe it didn't download that many subtitles at all. 

Is there a log somewhere to see what it did or didn't do? EDIT: Found it.

**Comment by bramwalet** [comment]
Yes a Subliminal plugin logfile should be located in the pms plugins directory. See https://support.plex.tv/hc/en-us/articles/200250417-Plex-Media-Server-Log-Files
